src/components/PDFView/index.js

```diff
@@ -17,22 +22,57 @@ const defaultProps = {
     style: {},
 };
 
-/**
- * On web, we use a WebView pointed to a pdf renderer
- *
- * @param props
- * @returns {JSX.Element}
- */
+class PDFView extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            numPages: null,
+        };
+        this.onDocumentLoadSuccess = this.onDocumentLoadSuccess.bind(this);
+        console.debug(props);
+    }
+
+    onDocumentLoadSuccess = ({numPages}) => {
+        this.setState({numPages});
+    }
+
+    render() {
+        const {isSmallScreenWidth, windowWidth} = this.props;
+        const pageWidth = isSmallScreenWidth ? windowWidth - 30 : variables.pdfPageWidth;
```

Comment:
Would it make sense to just always use `windowWidth - 30` as long as it is below some max value `variables.pdfPageWidth`? I think looking at whether we have the small screen just means that everything above that will be stuck at the constant width.

Reply:
`windowWidth - 30` only works on small screens as the is modal is full screen. But on the bigger screens, there is padding and margin around the pdf container so offset 30 would not work.

Reply:
I don't think it changes much about what I'm asking so I'll ask it another way in case my original question was not understood..

If we are above the small screen width we should set the `pageWidth` to equal `windowWidth` if it is less than `variables.pdfPageWidth`

Reply:
OK but then the page's width would be larger than the modal width as `windowWidth` is the width of the viewport. until the `variables.pdfPageWidth`.
So There are no specific variables for the modal offset from the edge and the `Page` does not take variable width so it's better to set `windowWidth - 100` which is 30(pdf edge gap) + 40(modal offset) + 30 (padding around).

Reply:
Right, makes sense. Mostly just want the PDF width to comfortably fit in the attachment modal at all times.